.devcontainer/devcontainer.json

```diff
@@ -1,9 +1,10 @@
 {
-  "build": { "dockerfile": "r-devel-gcc/Dockerfile", "context": ".." },
+  "build": { "dockerfile": "Dockerfile", "context": "../.." },
```

Comment:
it looks like you've still got some edits from earlier #6747 included here -- please revert, for example:

```shell
git checkout master -- .devcontainer/devcontainer.json
```